### DISCUSSION TITLE
Roadmap or considerations for macOS support and also extend beyond AVR8, like megaTinyCore?

### DISCUSSION THREAD

**Discussion by dattasaurabh82** [bloombloombloom/Bloom#99] [General]
I guess the one-liner question is self explanatory. 
Amazing project! Would love to see it grow ... 🙌🏻

**Comment by navnavnav** [comment]
Hello @dattasaurabh82 

Currently, there are no plans to support other operating systems and I don't see that changing in the foreseeable future.

I'm not sure what you mean by "extend beyond AVR8, like megaTinyCore". Bloom is oblivious to the program being debugged. It doesn't pay much attention to the Arduino "core" that may or may not be present in the program. It simply provides access to the target for debugging operations, via a GDB RSP server.
